src/index.css

```diff
@@ -10,5 +22,182 @@
   display: flex;
   flex-direction: row;
   width: 100%;
+  background-color: #fff;
+  border-radius: 5px;
+  border-color: #e8e8e8;
+  border-width: 1px;
+  border-style: solid;
+  height: 22vh;
+  margin-top: 2.375rem;
+  padding: 20px;
+}
+
+.category {
+  font-family: 'Montserrat', sans-serif;
+  color: #121212;
+  margin: 0 2.438rem 0 0;
+  font-size: 0.875rem;
+  opacity: 0.5;
+}
+
+.title {
+  font-size: 1.375rem;
+  width: 20vw;
+  font-family: 'Roboto Slab', serif;
+  color: #121212;
+}
+
+.author {
+  margin: 0.25rem 0.875rem 0 0;
+  font-family: 'RobotoSlab', serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  color: #4386bf;
+}
+
+.interact {
+  margin-top: 5%;
+}
+
+.interact-btn {
+  background-color: rgba(1, 1, 1, 0);
+  border: none;
+  margin: 0.188rem 0.938rem 0.125rem 0;
+  font-family: 'RobotoSlab', serif;
+  font-size: 0.875rem;
+  color: #4386bf;
+}
+
+.sep {
+  border-left-style: solid;
+  border-left-color: #e8e8e8;
+  border-left-width: 1px;
+}
+
+.left {
+  width: 25rem;
+}
```

Comment:
- [ ] As pointed out by the reviewer, there are still overflowing elements in this class element. Kindly fix them. :pray: 

![image](https://user-images.githubusercontent.com/64894927/135672787-91dbcfec-4705-4a60-a647-ec8c4ae6ec1c.png)